packages/opentelemetry-plugin-postgres/src/pg.ts

```diff
@@ -0,0 +1,160 @@
+/*!
+ * Copyright 2019, OpenTelemetry Authors
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import { BasePlugin } from '@opentelemetry/core';
+import { SpanKind } from '@opentelemetry/types';
+import { AttributeNames } from './enums';
+import { PostgresCallback, PostgresPluginOptions } from './types';
+import * as path from 'path';
+import * as pgTypes from 'pg';
+import * as shimmer from 'shimmer';
+
+export class PostgresPlugin extends BasePlugin<typeof pgTypes> {
+  static readonly component = 'pg';
```

Comment:
Should this be COMPONENT since it's immutable? Similar for SUPPORTED_VERIONS?

Reply:
Updated to COMPONENT. SUPPORTED_VERSIONS is specified by `BasePlugin` interface, so it should be changed there too. Maybe in a separate PR?